main/utils/blur.py

```diff
@@ -2,7 +2,7 @@
 import requests
 
 def getBlurCodeFromImage(imageUrl):
-        placeHolderImage = "/tmp/placeholder.jpg"
+        placeHolderImage = "./tmp/placeholder.jpg"
```

Comment:
`/tmp` is not recognized as a folder

Reply:
Might be purely a windows problem? While I'm here, you might consider using `pathlib`

Reply:
regarding this, I can change this out for a better solution in a new PR. /tmp is a temporary directory on Linux that doesn't persist after reboots, and I'm *relatively* sure that it's just a filesystem that's backed by RAM. It won't work on windows because /tmp doesn't exist.